sdk/formrecognizer/azure-ai-formrecognizer/swagger/v2.1-preview.3.json

```diff
@@ -18,11 +18,14 @@
     }
   ],
   "x-ms-parameterized-host": {
-    "hostTemplate": "{endpoint}/formrecognizer/v2.1-preview.3",
+    "hostTemplate": "{endpoint}/formrecognizer/{ApiVersion}",
```

Comment:
curious... so this only works because you have a copy of the swagger locally. What will be the case when you remove this copy?
There shouldn't be a need for this, right?